server/src/main/java/io/crate/execution/engine/aggregation/statistics/Variance.java

```diff
@@ -30,9 +30,9 @@
 
 public class Variance implements Writeable, Comparable<Variance> {
 
-    private double sumOfSqrs;
-    private double sum;
-    private long count;
+    protected double sumOfSqrs;
```

Comment:
Please revert also this, as currently, the change is irrelevant to the PR, it's usuful once the new stddev variation is added.